pkg/ccl/changefeedccl/schemafeed/schema_feed.go

```diff
@@ -65,14 +65,14 @@ type Config struct {
 	// change the table at ts1.Next() are emitted as an event.
 	InitialHighWater hlc.Timestamp
 
-	// LeaseManager is used to ensure that when an event is emitted that at a higher
+	// Manager is used to ensure that when an event is emitted that at a higher
```

Comment:
LeaseManager, not Manager (15th commit "rename types to remove stuttering" )

Reply:
Done.